core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -5698,7 +5698,7 @@ private void checkNegWindow(String s, String msg) {
             + " INTEGER DEPTNO,"
             + " BOOLEAN SLACKER,"
             + " INTEGER DEPTNO0,"
-            + " VARCHAR(10) NAME) NOT NULL");
+            + " VARCHAR(10) NAME)");
```

Comment:
I think that a structured type representing the result of a query does not make sense to be nullable (and cannot really be). I guess this change came from the fact that all returned fields are nullable. Maybe by introducing a separate field solves this problem. Same remark for the tests below.